docs/how-to/deploy_to_different_channels.md

```diff
@@ -0,0 +1,50 @@
+# Deploy your bot to different platforms
+
+To link a channel to your bot:
+
+!!! info "Note"
+    Not all channels require a provider.
+
+- Choose a [provider][2] for your channel.
+- [Configure a messaging provider][6]. You will need to get the required credentials from your chosen provider.
+- Once the provider is set up, navigate to your bot on OCS and click the plus icon in the "Channels" section.
+- Choose your channel and complete the form. Follow the guide below to get the required information for each channel.
+
+## Web
```

Comment:
maybe add API to this heading?

Reply:
Is [this](https://github.com/dimagi/open-chat-studio-docs/pull/20/commits/fa7fc6d317023597d169429aa67215e3b9e62a5a) what you mean?

Reply:
I didn't see the API header below this before - it seems fine the way it was before that commit (sorry).